cli/cmd/cache.go

```diff
@@ -25,7 +25,7 @@ import (
 	"time"
 
 	"github.com/lacework/go-sdk/internal/format"
-	homedir "github.com/mitchellh/go-homedir"
+	lwint "github.com/lacework/go-sdk/internal/lacework"
```

Comment:
lets not use `lwint` since it looks/sounds like lacework int = lacework integration 🤔 

It confused me a bit. I also don't know how to call this new package..... 🤔 `lacework` sounds too broad, but if we call it `lwcache`, then I don't know how `lwcache.FileExist()` would look like. 

I would suggest two packages:
* `/internal/cache` for cache stuff
* `/internal/file` for general file functions

We already have an internal package for `array` so it makes sense